CHANGELOG.md

```diff
@@ -28,6 +28,21 @@ All notable changes for each version of this project will be documented in this
     - `setOffset` method added. It offsets the content along the corresponding axis by the provided amount.
 - `IgxToggleDirective`:
     - `setOffset` method added. It offsets the content along the corresponding axis by the provided amount.
+- `IgxRowDragGhost` directive is added. It allows providing a custom template for the drag ghost when dragging a row.
+```html
+    <igx-grid #grid1 [data]="remote | async" [primaryKey]="'ProductID'"
+        [rowDraggable]="true">
+        <igx-column [field]="'ProductName'"></igx-column>
+        <igx-column [field]="'ProductID'"></igx-column>
+        <igx-column [field]="'UnitsInStock'"></igx-column>
+        <ng-template let-data igxRowDragGhost>
+            <div>
```

Comment:
Just curious - is the div required?

Reply:
Not really.

Reply:
Actually, the icon was redundant. However, we need the div for the correct behavior.